src/applications/personalization/profile360/containers/PaymentInformation.jsx

```diff
@@ -128,26 +128,24 @@ class PaymentInformation extends React.Component {
 
   handleDirectDepositUpdateSubmit = data => {
     this.props.savePaymentInformation(data);
-    recordEvent({
-      event: 'profile-transaction',
-      'profile-section': 'direct-deposit-information',
-    });
   };
 
-  handleEditClick = gaProfileSection => {
-    // Open edit modal.
+  handleLinkClick = (linkType, gaProfileSection) => {
```

Comment:
Might I suggest some ideas for making this a little better, IMHO? Because I can't help but err on the side of over-engineering :-) 

First thought: you could store the two possible `linkType`s in constants (I hate throwing strings around). And if you do that you could create a simple map of `linkType` to `profile-action`. So that you don't have the inline ternary on line 139.

Second idea: change the new `linkType` arg from a string to a bool, since we are only dealing with `edit` or `add`. Then line 139 could be rewritten to something like `isAddLink ? 'add-link' : 'edit-link'`